test/swar/BasicOperations.cpp

```diff
@@ -455,3 +456,36 @@ TEST_CASE(
     CHECK(SWAR<4, u16>(0x0F00).value() == saturatingUnsignedAddition(SWAR<4, u16>(0x0800), SWAR<4, u16>(0x0800)).value()); 
     CHECK(S4_32(0x0F0C'F000).value() == saturatingUnsignedAddition(S4_32(0x0804'F000), S4_32(0x0808'F000)).value()); 
 }
+
+TEST_CASE(
+    "cheapNullChecking",
+    "[swar][unsigned-swar]"
```

Comment:
What is the meaning of the tag "unsigned-swar"?

Reply:
I left it as I was testing with unsigneds, but you feel a lot more strongly about tags than I do.

Reply:
I want us to keep using tags so that eventually we converge into a good system of tagging.
I object to the name of the test.